dist/t/spec/features/0040_package_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+
+RSpec.describe "Package" do
+
+  it "should be able to create new" do
+    login('Admin','opensuse')
+    within("div#subheader") do
+      click_link('Home Project')
+    end
+    expect(current_path).to eq '/project/show/home:Admin'
+    click_link('Create package')
+    fill_in 'name', with: 'ctris'
+    fill_in 'title', with: 'ctris'
+    fill_in 'description', with: 'ctris'
+    click_button('Save changes')
+    expect(page).to have_content("Package 'ctris' was created successfully")
+
+    logout
+  end
+
+  it "should be able to upload files" do
```

Comment:
What about also testing the interconnect? So branching from interconnect? However, then we need to rely that build.o.o is working and that also the package we branch is not broken ...